### DISCUSSION TITLE
Putty

### DISCUSSION THREAD

**Discussion by palsachennaiah** [learndevopsonline/learndevopsonline#1594] [Q&A] [answered]
![Screenshot 2023-11-11 102451](https://github.com/learndevopsonline/learndevopsonline/assets/148023299/85aa884d-3f8f-4a90-905a-0b197ddce6ce)
My putty is not working

**Comment by sreedharm07** [comment]
Attach allow all security group to instance

**Reply by palsachennaiah** [reply]
Every thing i have done but its not working

**Reply by sreedharm07** [reply]
Delete instance create new

**Reply by palsachennaiah** [reply]
No clue I got it

**Reply by palsachennaiah** [reply]
![Screenshot 2023-11-11 152034](https://github.com/learndevopsonline/learndevopsonline/assets/148023299/1e95e349-f784-4b67-b64e-045aba25e382)
Hi say something about putty its not working, three instances changed, putty uninstalled & reinstalled even though same thing happened, what is the problem of putty any one respond.

**Reply by sreedharm07** [reply, marked as answer]
check list -
-- created centos 8 ami
-- selected no key pair
-- attached allow all security group
-- that allow all security in bound rules should be all traffic

if you given all this, and even its not working means please join lab quary session. because it cont be figured out here.

**Reply by sreedharm07** [reply]
i tried in another terminal (mobaxterm) its even not working there so its  not about putty and its fault in instance configuration only

**Comment by palsachennaiah** [comment]
![Screenshot 2023-11-11 104427](https://github.com/learndevopsonline/learndevopsonline/assets/148023299/7941928c-d111-4d87-9e66-4f6e18b42bb2)

**Reply by palsachennaiah** [reply]
That is also, i did

**Reply by palsachennaiah** [reply]
please you do check it with my IP address 3.92.22.128
